src/resizable-card-layout.scss

```diff
@@ -0,0 +1,282 @@
+@import './new-settings';
+@import './mixins';
+@import './icon';
+
+$block: #{$fd-namespace}-resizable-card-layout;
+$blockContainer: #{$fd-namespace}-card-container;
```

Comment:
I am not sure if this is okay considering that PR checklist asks for : `only one top level fd-* class is used in the file`